bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -634,7 +634,7 @@ public void asyncReadUnconfirmedEntries(long firstEntry, long lastEntry, ReadCal
      *          id of last entry of sequence
      */
     @Override
-    public CompletableFuture<Iterable<org.apache.bookkeeper.client.api.LedgerEntry>> read(long firstEntry, long lastEntry) {
+    public CompletableFuture<org.apache.bookkeeper.client.api.LedgerEntries> read(long firstEntry, long lastEntry) {
```

Comment:
nit: I don't think you need fully qualified path name. it can just be `CompletableFuture<LedgerEntries>`.

Reply:
Thanks, will change it.